server/src/test/java/org/elasticsearch/cluster/metadata/MetadataCreateDataStreamServiceTests.java

```diff
@@ -396,7 +461,7 @@ public void testCreateDataStreamNoValidTemplate() throws Exception {
         );
     }
 
-    public static ClusterState createDataStream(final String dataStreamName) throws Exception {
+    public static ClusterState createDataStream(final String dataStreamName, ThreadPool threadPool) throws Exception {
```

Comment:
The `threadpool` here is not necessary anymore right?

Reply:
Ah this is sloppy sorry...